explainer.md

```diff
@@ -0,0 +1,793 @@
+# Temporal
+
+## Timeline Objects
+
+There is a subset of temporal objects that are tied to the absolute timeline. They represent a specific point in time.
+
+These are `Instant`, `OffsetDateTime` and `ZonedDateTime` in order of amount of detail information available in them.
+
+Instant just represents an absolute point in time. No timezone or offset information is present. As such Intants have no concept of days or months or even hours.
+
+OffsetDateTime is the combination of an Instant with an offset from UTC. As such it has the ability to know about days, months, etc. However it does not know what timezone or locality it is in. As such it cannot know which daylight saving rules apply.
+
+ZonedDateTime represents an OffsetDateTime combined with an *IANA Timezone*. With that added information ZonedDateTime is able to observe daylight savings rules.
+
+### Instant
+An `Instant` is an object that specifies a specific point in time. For convenience of interoperatbility it uses *nanoseconds since the unix-epoch* to do so.
```

Comment:
```suggestion
An `Instant` is an object that specifies a specific point in time. For convenience of interoperability it uses *nanoseconds since the unix-epoch* to do so.
```